src/Controls/samples/Controls.Sample/Pages/MainPage.cs

```diff
@@ -227,7 +227,7 @@ void SetupMauiLayout()
 			var entryMargin = new Thickness(10, 0);
 
 			verticalStack.Add(entry);
-			verticalStack.Add(new Entry { Text = "Entry", TextColor = Colors.DarkRed, FontFamily = "Dokdo", MaxLength = -1, Margin = entryMargin });
+			verticalStack.Add(new Entry { Text = "Entry with custom Font", TextColor = Colors.DarkRed, FontFamily = "Dokdo", MaxLength = -1, Margin = entryMargin });
```

Comment:
What is MaxLength=-1 expected to do?

Reply:
Is TextColor not yet expected to work?

Reply:
TextColor is still not implemented on Windows. MaxLength=-1, simply does not limit the characters, that value is used to test that there are no errors (using a negative value).

Reply:
```suggestion
			verticalStack.Add(new Entry { Text = "Entry with custom Font", TextColor = Colors.DarkRed, FontFamily = "Dokdo", MaxLength = -1, Margin = entryMargin });
```
```suggestion
			verticalStack.Add(new Entry { Text = "Entry with custom Font", TextColor = Colors.DarkRed, FontFamily = "Dokdo", MaxLength = -1, Margin = entryMargin });
```